.travis.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         - go mod vendor
       script:
         # Run multinamespace tests as admin (skip role settings check)
-        - make test PARALLEL_COUNT=2 && kubectl config use-context $TESTING_CONTEXT && make multinamespace-test
+        - make test PARALLEL_COUNT=2 && kubectl config use-context $TESTING_CONTEXT && make batch-test && make multinamespace-test
```

Comment:
@ryanemerson do you need to run `batch-test` without specific role with admin account? If not, this should run before `kubectl config use-context $TESTING_CONTEXT`

Reply:
Updated